common/src/main/java/org/apache/gravitino/credential/CredentialPropertyUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.gravitino.credential;
+
+import java.util.Map;
+
+/**
+ * Helper class to generate specific credential properties for different table format and engine.
+ */
+public class CredentialPropertyUtils {
+  /**
+   * Transforms a specific credential into a map of Iceberg properties.
+   *
+   * @param credential the credential to be transformed into Iceberg properties
+   * @return a map of Iceberg properties derived from the credential
+   */
+  public static Map<String, String> toIcebergProperties(Credential credential) {
+    // todo: transform specific credential to iceberg properties
+    return credential.toProperties();
```

Comment:
This two lines code is not worthy to create a new class here, why can't you put this in iceberg package?

Reply:
it's like a placeholder function to support more credential properties transform logic, this function will be used in client side and Iceberg REST server side, so I place it in common module.